strawberryfields/backends/base.py

```diff
@@ -222,13 +222,16 @@ def add(self, num_modes):
 class BaseBackend:
     """Abstract base class for backends."""
 
+    #: str: short name of the backend
+    short_name = 'base'
+    #: str, None: Short name of the CircuitSpecs class used to validate Programs for this backend. None if no validation is required.
+    circuit_spec = None
```

Comment:
Awesome

Reply:
With the new `GBSSpec`, the backends now support multiple CircuitSpecs for validation. I suppose `fock` and `gaussian` are the _backend specific_ specifications - even if the user compiles againsts `GBSSpec`, it will still need to be validated by the backend against `Backend.circuit_spec` to ensure it can be correctly executed.